### DISCUSSION TITLE
Provide a personal collection to users

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#4000] [Ideas]
Hello there

I find useful to be able to let people to have their own collection for their personal documents (writing their tips for work), it permit also to let people write documentation until it is ready for a review, or until they find a proper place, so after they can move it where it fits to a group collection. I used to do that a lot in confluence and I miss that in outline. 

Best.

**Comment by tommoor** [comment]
Yes, this would be nice – we've spiked it out before but it deserves to be a feature. In the meantime of course you can create a private collection that only you have access to and star it so that it appears at the top of the sidebar. This should act much the same 😄
